web_view_google_map/views/res_partner.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_res_partner_google_map" model="ir.ui.view">
+        <field name="name">view.res.partner.google_map</field>
+        <field name="model">res.partner</field>
+        <field name="arch" type="xml">
+            <google_map class="o_res_partner_map" library="geometry" string="Google Map" lat="partner_latitude" lng="partner_longitude" colors="blue:company_type=='person';green:company_type=='company';">
```

Comment:
@brian10048 , 
I don't see any js widget for a google_map tag

Reply:
Thank you for noticing @AmmarOfficewalaSerpentCS 
I updated the JS files for the change from `map` to `google_map`